doc/using/overlays.chapter.md

```diff
@@ -51,15 +51,15 @@ self: super:
   boost = super.boost.override {
     python = self.python3;
   };
-  rr = super.callPackage ./pkgs/rr {
+  rr = self.callPackage ./pkgs/rr {
     stdenv = self.stdenv_32bit;
   };
```

Comment:
```suggestion
```

Let's avoid the `callPackage` example entirely for now, since it's a bit confusing (both `self.callPackage` and `super.callPackage` use dependencies from `self`).

Reply:
I left the `callPackage` in originally as a demonstration of pulling a function from `self`. Is there a less confusing example that we can use, perhaps a fetcher (like the `xz` example)?

Reply:
Yeah a fetcher sounds pretty good :)